public/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+    "theme_color": "#101010",
+    "background_color": "#f8e71c",
+    "display": "standalone",
+    "scope": "/",
+    "start_url": "/",
+    "name": "Stacker News",
+    "short_name": "SN",
+    "description": "Your favourite news sources",
```

Comment:
Maybe this should use the default SN description?

`It's like Hacker News, but we pay you Bitcoin`